src/librehatti/settings.py

```diff
@@ -19,9 +19,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'db_name',
-        'USER': 'db_user',
-        'PASSWORD': 'db_password',
+        'NAME': 'lh',#'db_name',
```

Comment:
why the name, password are here?

Reply:
It's my mistake. Thanks for pointing it out. @mandeeps708 Should I do one more commit in order to correct it as there are already many commits? Or you are having some better solution.

Reply:
Do one more commit as This whole will be eventually squeezed to one